ValerieRono/ireporter
# #162323944 edit specific record

#### What does this PR do?
allow a user to edit any field of a specific record

#### Description of Task to be completed?
create put method
add method to resource

#### How should this be manually tested?
follow instructions in the readme file and ensure app is running
using postman set method to put
copy url from the terminal(refer to read me), insert /incidents/<int:id> where <int:id> is the id of the record you would like to edit after the url
enter key/value pair of fields you want to edit in the body
press send

#### Any background context you want to provide?
used flask marshmallow and flask restful

#### What are the relevant pivotal tracker stories?
#162323944

#### Screenshots (if appropriate)
![screenshot from 2018-11-30 10-29-54](https://user-images.githubusercontent.com/40663828/49274952-eb96e880-f48a-11e8-9fba-4d8634cda45a.png)
